virtool/fake/next.py

```diff
@@ -156,3 +160,36 @@ async def create(
             return await self.layer.users.get(user.id)
 
         return user
+
+
+class HMMFakerPiece(DataFakerPiece):
+    model = HMM
+
+    async def create(self, mongo):
+
+        hmm_id = self.faker.pystr()
+        faker = self.faker
+
+        await mongo.hmm.insert_one(
```

Comment:
I removed the second query, although I'm not sure how I can use `find_one_and_update` because there would not yet be a document in mongo to find / update.

Reply:
Right. Sorry, that was a brain fart. The inserted document should be returned by `insert_one`.